lib/elixir/lib/module/parallel_checker.ex

```diff
@@ -329,27 +329,39 @@ defmodule Module.ParallelChecker do
     ["  ", Exception.format_stacktrace_entry(stacktrace), ?\n]
   end
 
-  defp to_diagnostic(message, {file, line, mfa}) do
+  defp to_diagnostic(message, {file, line, mfa}) when is_integer(line) do
```

Comment:
No problem, will there be anything else to test without these changes? Since they are only changing compiler internals, I guess just to have the test suite passing is enough, right?